### PR TITLE
[Support] Android show/hide view layers button

### PR DESCRIPTION
**Pull request by eamon-mallon**
Ticket: https://support.pdftron.com/a/tickets/23262
- Added new constant BUTTON_VIEW_LAYERS_TOOLBAR to android
- Updated DocumentView to handle BUTTON_VIEW_LAYERS_TOOLBAR
- Added showViewLayersToolbarButton to Config.js

### CODE REVIEW

File: android/src/main/java/com/pdftron/reactnative/utils/Constants.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
     public static final String BUTTON_THUMBNAIL_SLIDER = "thumbnailSlider";
     public static final String BUTTON_SAVE_COPY = "saveCopyButton";
     public static final String BUTTON_EDIT_PAGES = "editPagesButton";
+    public static final String BUTTON_VIEW_LAYERS_TOOLBAR = "showViewLayersToolbarButton";
```
Comment (sgong-pdftron) [inline android/src/main/java/com/pdftron/reactnative/utils/Constants.java:35]: "showViewLayers"

File: src/Config/Config.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ export default {
     searchButton: 'searchButton',
     shareButton: 'shareButton',
     editPagesButton: 'editPagesButton',
+    showViewLayersToolbarButton: 'showViewLayersToolbarButton',
```
Comment (sgong-pdftron) [inline src/Config/Config.js:28]: showViewLayers

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: src/Config/Config.js (hunk 2 above)
Comment (eamon-mallon) [reply src/Config/Config.js:28]: Does that not break the naming convention of the other options ?

Comment (sgong-pdftron) [reply src/Config/Config.js:28]: good point, showViewLayersButton

File: android/src/main/java/com/pdftron/reactnative/utils/Constants.java (hunk 1 above)
Comment (sgong-pdftron) [reply android/src/main/java/com/pdftron/reactnative/utils/Constants.java:35]: showViewLayersButton

**Comment by sgong-pdftron** [comment]
`viewLayersButton` after seen adrienne's

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**